disttask/framework/dispatcher/dispatcher_test.go

```diff
@@ -144,7 +143,7 @@ func checkDispatch(t *testing.T, taskCnt int, isSucc bool) {
 		dsp.Stop()
 		// make data race happy
 		if taskCnt == 1 {
-			dispatcher.DefaultDispatchConcurrency = originalConcurrency
```

Comment:
I call this line after `dsp.Stop()`, it seems this won't encounter data race.  Or is it something else?